Mathlib/Data/Quot.lean

```diff
@@ -693,7 +694,7 @@ theorem hrecOn₂'_mk'' {φ : Quotient s₁ → Quotient s₂ → Sort*}
 
 /-- Map a function `f : α → β` that sends equivalent elements to equivalent elements
 to a function `Quotient sa → Quotient sb`. Useful to define unary operations on quotients. -/
-protected def map' (f : α → β) (h : (s₁.r ⇒ s₂.r) f f) : Quotient s₁ → Quotient s₂ :=
+protected def map' (f : α → β) (h : ∀ a b, a ≈ b → f a ≈ f b) : Quotient s₁ → Quotient s₂ :=
```

Comment:
This very deliberately uses `.r` and not `≈`; that's the purpose of the `'` in the name.

Reply:
gotcha @eric-wieser, makes sense that it does - I've changed it accordingly. I don't see how that's obvious from the `'` in the name to be frank.